src/CommandingOfficers/AW4/RuinedCommander.java

```diff
@@ -59,7 +59,9 @@ public abstract class RuinedCommander extends DeployableCommander
 
   public static final int GENERIC_STAT_BUFF = 10;
   /** If the unit type matches any flag in this mask, it receives my zone stat boosts */
-  public long canBoostMask = Long.MAX_VALUE;
+  public long boostMaskAny = Long.MAX_VALUE;
+  /** Boost if all flags match; you should customize boostMaskAny if you use it */
+  public long boostMaskAll = Long.MAX_VALUE;
```

Comment:
Looks like you'd also have to set `boostMaskAny` to zero if using `boostMaskAll`. Oh, that's probably what you are saying. It wasn't clear what "it" refers back to, especially since the comment is on `boostMaskAll`. Changing "it" to "this" at a minimum (and add "also" before "customize") might make it more clear, but it also wouldn't be out of line to clarify more (or at least add a comment to `boostMaskAny`; clarify that it boosts everything by default, and that both conditions are OR'd to get the final boost mask.